pkg/config/manager.go

```diff
@@ -227,3 +228,7 @@ func (c *Manager) Start() {
 	c.ticker = time.NewTicker(c.loopDelay)
 	go c.loop()
 }
+
+func init() {
```

Comment:
Can we put all the different storage backends in an `inventory()` similar to the `/adapter/inventory` instead?